core/src/test/java/org/frankframework/receivers/MockPullingListener.java

```diff
@@ -33,7 +33,7 @@ public RawMessageWrapper<String> getRawMessage(Map<String, Object> threadContext
 			if("getRawMessageException".equals(message)) {
 				throw new ListenerException(message);
 			}
-			return new RawMessageWrapper<>(message);
+			return new RawMessageWrapper<>(message, Integer.toString(message.hashCode()), Integer.toString(message.hashCode()));
```

Comment:
Messages sent to PullingListenerContainer need to have an ID, otherwise assumptions in the code break.
These assumptions were implicit but I've now made them explicit, hence test needs to send messages with ID.